testnet/stacks-node/src/config.rs

```diff
@@ -30,17 +29,28 @@ pub struct ConfigFile {
     pub block_limit: Option<BlockLimitFile>,
 }
 
+#[derive(Clone, Deserialize, Default)]
+pub struct LegacyMstxConfigFile {
```

Comment:
I don't think we need to worry about legacy support -- we're pre-release still.  Maybe we can just do a `panic!()` if we see `mstx_balance` so users know to change it to `ustx_balance`.

Reply:
I think we should leave it for now to make things easier for devs. We can remove after next marge to master.